.ci/cico_rhche_compatibility_test.sh

```diff
@@ -36,25 +79,53 @@ checkAllCreds
 echo "Installing dependencies:"
 installDependencies
 
+echo "Configuring remote:"
+git remote set-url origin https://${GITHUB_AUTH_STRING}@github.com/redhat-developer/rh-che.git
+
 #Check branch for tracking changes/Create new branch
 BRANCH="upstream-check-$CHE_VERSION"
 echo "Checking if branch $BRANCH exists."
 set +e
-git checkout "$BRANCH"
+git checkout "$BRANCH" 2>&1 >/dev/null
 return_code=$?
 set -e
-if [ $return_code -eq 0 ]; then 
-  echo "Branch $BRANCH found - rebasing."
-  git rebase origin/master "$BRANCH"
-else 
+if [ ! $return_code -eq 0 ]; then
   echo "Branch $BRANCH not found - creating new one."
+  git checkout master
   git checkout -b "$BRANCH"
-	
-  #change version of used che
+  echo "Setting new branch origin"
+  git push --set-upstream origin $BRANCH
   echo ">>> change upstream version to: $CHE_VERSION"
   scl enable rh-maven33 "mvn versions:update-parent  versions:commit -DallowSnapshots=true -DparentVersion=[${CHE_VERSION}] -U"
 fi
 
+if ( git diff --exit-code ); then
+  echo "Nothing to commit, continue."
+else
+  echo "Changes found. Commit and push them before creating PR."
+  git add -u
+  git commit -m "Changing version of parent che to $CHE_VERSION" || echo "No changes found to commit."
+  git push origin "$BRANCH" -f
+  if [[ "$?" != "0" ]]; then
+    echo $? > compatibility_test
```

Comment:
Do we want to continue when branch is not pushed?

Reply:
same as https://github.com/redhat-developer/rh-che/pull/1647#discussion_r332447246